drivers/nutdrv_qx.c

```diff
@@ -928,7 +935,7 @@ static int	krauler_command(const char *cmd, char *buf, size_t buflen)
 
 		int	retry;
 
-		if (strcmp(cmd, command[i].str)) {
+		if (strncmp(cmd, command[i].str, cmdlen)) {
```

Comment:
Sanity-checking this change here and in many other routines: should it be `strncmp` to only compare the first N characters of the `cmd`? By the new argument, seems so, but still...

Avoiding an unintended `strncmp("shutdown", "shut up!", 4) == true` sort of mishap ;)